website/templates/public/pages/help/files.mako

```diff
@@ -7,7 +7,7 @@
     <p>To upload a file, click on "Files" in the grey navigation bar under the Project/Component's title. Here you
         can drag the file from your desktop on to the screen to upload, or click on the upload button in the actions
         column. To download a file from a project, click the download button in the actions column. You can delete
-        files by clicking the 'X' that appears when you hover over the file.</p>
+        files by clicking the 'X' that appears when you hover over the file. You can create folders to organize files by clicking the + button to the right of the upload button.</p>
```

Comment:
Instead of saying `'X'`, and '+', you can use actual icons in the markup instead, e.g. `<i class='fa fa-plus'></i>`.
-----------
![osf___getting_started](https://cloud.githubusercontent.com/assets/2379650/7566228/a03c888a-f7c3-11e4-8fd9-c09596964875.png)
------

I'll make this change when I merge.